docs/workflow/building/coreclr/linux-instructions.md

```diff
@@ -19,7 +19,7 @@ Please note that when choosing an image choosing the same image as the host os y
 Once you have chosen an image the build is one command run from the root of the runtime repository:
 
 ```sh
-docker run --rm -v <RUNTIME_REPO_PATH>:/runtime -w /runtime mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-a50a721-20191120200116 ./src/coreclr/build.sh -clang9
+docker run --rm -v <RUNTIME_REPO_PATH>:/runtime -w /runtime mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-a50a721-20191120200116 ./build.sh --subset clr -clang9
```

Comment:
did the root build.sh work for you with that specific image? I was seeing a dependency failure when I tried it last.

Reply:
The latest image tags are listed here:
https://github.com/dotnet/versions/blob/master/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-master.json
It would be the best to use those.
So for ubuntu-16.04, the most recent tag is `ubuntu-16.04-20200508132555-78cbb55`.

Reply:
Yes that worked for me. I'll update the tags to latest you recommend Jan, thanks for the pointer. I tested that tag works fine with this `docker run` command line.

Reply:
I double-checked both commands compile successfully with the updated tags 👍